moc_sprint_tools/create_sprint_boards.py

```diff
@@ -10,7 +10,7 @@
 
 CARD_COPYING_MAP = {
     'notes': 'notes',
-    'backlog': 'backlog',
+    'sprint backlog': 'sprint backlog',
```

Comment:
It turns out that existing boards are using the column name "sprint backlog" so I updated this here (and elsewhere).